spec/core/cbp_helper.rb

```diff
@@ -0,0 +1,21 @@
+def expect_cbp_response_for(collection)
+  context collection.path.to_s, :vcr do
+    before do
+      @resource_klass = collection.instance_variable_get(:@resource_class)
+      VCR.use_cassette("cbp_#{@resource_class}_#{collection.path}") do
+        @result = collection.fetch
+        @response_body = collection.response.body
+      end
+    end
+    it 'expects an array with the correct element types' do
```

Comment:
Maybe?

```suggestion
    it "returns a list of #{@resource_klass}" do
```

I think in this context it should be available, you're inside a method....

However, I'm not a fan of extracting whole tests in methods. But I can't verbalize that, so maybe it's a subjective thing. I'm ok with a bit of duplication in the code, or using a list of test params like we do in Go.

The context could also use some more wording.

Lastly, I see the context, but I don't see what are we describing?

Reply:
Done it ✅ 
also using the described_class var provided by rspec, much nicer